code/game/gamemodes/dynamic/antag_rulesets.dm

```diff
@@ -0,0 +1,171 @@
+#define MINDSHIELDED_JOBS list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Blueshield", "Nanotrasen Representative", "Magistrate", "Internal Affairs Agent", "Nanotrasen Navy Officer", "Special Operations Officer", "Trans-Solar Federation General")
+
+/datum/ruleset
+	/// What this ruleset is called
+	var/name = "BASE RULESET"
+	/// The cost to roll this ruleset
+	var/ruleset_cost = 1
+	/// The weight to roll this ruleset
+	var/ruleset_weight = 1
+	/// The cost to roll an antagonist of this ruleset
+	var/cost = 1
+	/// The weight to roll an antagonist of this ruleset
+	var/weight = 1
+	/// Antagonist datum to apply to users
+	var/datum/antagonist/antagonist
```

Comment:
i don't know what are rules are about this but if a var only holds a type, i prefer its name to be suffixed e.g. `antagonist_type`. I think typically we wouldn't also cast vars which are just types but I imagine we have to here to take advantage of e.g. `antagonist::job_rank` syntax? would like @Burzah to chime in for style guidance